src/applications/personalization/preferences/reducers/index.js

```diff
@@ -77,6 +80,7 @@ export default function preferences(state = initialState, action) {
       return {
         ...state,
         availableBenefits,
+        availableBenefitsCached: true,
```

Comment:
suggest tracking something like `lastLoadTime` and basing the caching behavior off of a time diff rather than just a boolean.

Reply:
'Based on a time diff', as in after a certain time we should re-fetch the latest data from the server? The only time I can imagine the local version going stale is in the extreme edge case that a user is in the middle of a session in one browser and starts another session in a different browser. Unless I'm missing a particular case or reason we'd want to hit the server again I think I'll stick with the current implementation.